buildCompleteWebsite.sh

```diff
@@ -26,31 +26,31 @@ mkdir -p build
 cd build
 if [ -d "xeps" ]; then
 	cd xeps
-	git pull
+	git pull || exit 1;
```

Comment:
I've heard `set -e` breaks if you use if-statements that are based on exit code of programs. However I'm very bad at bash scripting.